Feature/use bootstrap toast

sesamechicken opened the pull request:
- removed notify.min.js
- committed `package-lock.json` for faster dep resolution with npm
- added `.DS_Store` to `.gitignore`
- added nodemon to `devDependencies` and `dev` script entry in package.json for easier dev experience
- updated Bootstrap version
- updated jQuery version
- loaded jQuery before Bootstrap for proper init
- spec'd modern versions of node and npm in package.json

Commit:
Uses Bootstrap toast now, added nodemon for development (npm run dev), updated Bootstrap version, updated jQuery version, loaded jQuery before Bootstrap for proper init

Commit:
Uses Bootstrap toast now, added nodemon for development (npm run dev), updated Bootstrap version, updated jQuery version, loaded jQuery before Bootstrap for proper init

Commit:
Removed commented html, updated package.json to reflect a modern version of node and npm

Commit:
Removed commented html, updated package.json to reflect a modern version of node and npm

Commit:
Removed console.log

Commit:
Removed console.log

sesamechicken commented:
Addresses #121

LalitNM commented:
Your work looks good, but we want following changes in this pull request.

- Do not update `.gitignore` with just `.DS_Store`, as this needs much more that just `.DS_Store`(I will update this with all required filenames)

- Do not commit `package.json` and `package-lock.json` I will update this in a few days, until then keep them as local files, I will update them according to requirements of this project.

- Do not use custom styling for toast, use default styling used in bootstrap, see [this](https://getbootstrap.com/docs/4.2/components/toasts/) link

- After adding a new room this add produces a bug, you can not navigate back to `Lobby` room and cannot make a new room also, screenshot: 
![image](https://user-images.githubusercontent.com/63332774/95758579-2e859200-0c98-11eb-8b65-690b138e4ab0.png)

Fix the bug, because with this I can't test other features of notification.

Update this pull request, then I will again review it.

LalitNM committed:
Update to .gitignore

Changes
- added .gitignore from templates provided by github
- added .DS_Store for macOS

sesamechicken commented:
Whether you update the gitignore in a couple of days or not, there’s no
reason not to add to it in the mean time. Same with package.json. The
lockfile should exist - none of those files should be local.

There is no error status for Bootstrap’s toast, so I was sensibly
replicating what you had by adding a text color for your ‘error’ state.

I’ll take a look at the bug.

Thx

On Mon, Oct 12, 2020 at 09:38 Lalit Narayan Mudgal <notifications@github.com>
wrote:

> Your work looks good, but we want following changes in this pull request.
>
>    -
>
>    Do not update .gitignore with just .DS_Store, as this needs much more
>    that just .DS_Store(I will update this with all required filenames)
>    -
>
>    Do not commit package.json and package-lock.json I will update this in
>    a few days, until then keep them as local files, I will update them
>    according to requirements of this project.
>    -
>
>    Do not use custom styling for toast, use default styling used in
>    bootstrap, see this
>    <https://getbootstrap.com/docs/4.2/components/toasts/> link
>    -
>
>    After adding a new room this add produces a bug, you can not navigate
>    back to Lobby room and cannot make a new room also, screenshot:
>    [image: image]
>    <https://user-images.githubusercontent.com/63332774/95758579-2e859200-0c98-11eb-8b65-690b138e4ab0.png>
>
> Fix the bug, because with this I can't test other features of notification.
>
> Update this pull request, then I will again review it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/KamandPrompt/baat-cheet/pull/126#issuecomment-707160501>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJP7IMS66BLBLWCEPB7WLDSKMIE3ANCNFSM4SM2TDDQ>
> .
>
-- 
Sent from my iPhone

LalitNM committed:
Fixed vulnerabilities

Changes
- update to package.json

LalitNM commented:
I understand your issue @sesamechicken, that is why I immediately updated all files, now you are good to go. I am also resolving conflicts :slightly_smiling_face:

LalitNM committed:
Merge branch 'master' into feature/use-bootstrap-toast

namrataaaa committed:
Added the bootstrap modal for creating a new room.  (#125)

Changes

- Update to client.js in order to function modal properly
- Removed unnecessary code from script.jd
- Removed unnecessary styling from style.css and added styling for modal-body and header
- Added cdn for bootstrap in head_imports.ejs
- Update to index.js to add modal


Co-authored-by: Lalit Narayan Mudgal <63332774+LalitNM@users.noreply.github.com>

chris-kt committed:
Fixed merge conflicts

sesamechicken commented:
@LalitNM I'm not able to reproduce the bug you're describing, nor am I able to login to the Zulip chat app to ask questions. (says my email is not whitelisted)

LalitNM commented:
Thanks @sesamechicken for making me aware that you cannot join us on zulip, but you can join our community on gitter, where we can discuss. Link to the gitter community is [here](https://gitter.im/Baat-Cheet-IIT/community) and also in readme.md.

LalitNM commented:
Continued in #130.
